lib/Mojolicious/Plugin/AssetPack/Pipe/Export.pm

```diff
@@ -0,0 +1,92 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::Export;
+
+use Mojo::Base 'Mojolicious::Plugin::AssetPack::Pipe';
+use Mojo::Util 'spurt';
+
+use File::Basename 'dirname';
+use File::Path 'make_path';
+use File::Spec;
+
+has 'export_dir';
```

Comment:
Do this instead:

    has export_dir => sub { $ENV{MOJO_ASSETPACK_EXPORT_DIR} || '' };